app/views/sentences/show_all_in.ctp

```diff
@@ -27,18 +27,28 @@
 
 $languageName = $languages->codeToNameToFormat($lang);
 
-$title = format(__('All sentences in {language}', true),
-                array('language' => $languageName));
+if ( $filterAudioOnly == 'Yes' ) {
+    $title = format(__('{language} sentences with audio', true),
+        	    array('language' => $languageName));
+} else {
+    $title = format(__('{language} sentences', true),
+        	    array('language' => $languageName));
+}
 if (!empty($notTranslatedInto) && $notTranslatedInto != 'none') {
     if ($notTranslatedInto == 'und') {
         $notTranslatedIntoName = __('any language', true);
     } else {
         $notTranslatedIntoName = $languages->codeToNameToFormat($notTranslatedInto);
     }
-    $title = format(
-        __('Sentences in {language} not translated into {translationLanguage}', true),
-        array('language' => $languageName, 'translationLanguage' => $notTranslatedIntoName)
-    );
+    if ( $filterAudioOnly == 'Yes' ) {
```

Comment:
Same comment here. Also, please try to follow the code style of the existing code when adding yours:
```php
if ( a ) { # not OK
if (a) { # OK
```